### PR TITLE
Initial commit of mobile screen templates

### PR DESCRIPTION
**Pull request by janmenjoyroy**
@c-bik
Screen templates for mobile browser support. 
Screens provided for the following:
1. Home Page or the very first launching page
2. Services page
3. Services ---> Consultation page
4. Services menu page
5. 3. Services ---> Consultation menu page
6. Companies ---> Location map page
7. Companies ---> Team/partner page
8. Companies ---> Team/partner Details page

### CODE REVIEW

**Review by c-bik — CHANGES_REQUESTED** [review]
Use npm for bootstrap and font awesome libraries (js+css). Not a copy of 3rd party packages in our repo!

Refer to sample in development branch as how it should be done:
https://github.com/K2InformaticsGmbH/k2informaticsgmbh.github.io/blob/d9dfc01157863711ff20b807a26bcb397dd2c490/dev/package.json#L40-L46
https://github.com/K2InformaticsGmbH/k2informaticsgmbh.github.io/blob/d9dfc01157863711ff20b807a26bcb397dd2c490/dev/index.js#L1-L4

Ideally other than HTML, custom scss etc only package.json changes are desired!

**Review by c-bik — APPROVED** [review]

**Review by c-bik — CHANGES_REQUESTED** [review]
the **public** produced by `npm run build` is missing files:
image

I could view the work-in-progress with `npm start`! It looks like a good start with responsive in a few decives (checked with chrome **Device Toolbar**).

**To collect feedback from others, we need to be able to generate the public folder to host at https://k2informaticsgmbh.github.io/ !** - _please see fix this_

**Review by c-bik — APPROVED** [review]

**Review by shamis — COMMENTED** [review]
- Title and fav icon is not set i think
- Footer on larger screens are not on the foot of the page
- Services and Solutions link on the page does not go the header services and solutions links

**Review by c-bik — CHANGES_REQUESTED** [review]
@janmenjoyroy Unfortunately I could not compile to check your last commit.

### Fix package vulnarebilities
```sh
$ npm install
...
added 1618 packages from 903 contributors and audited 17698 packages in 186.356s
found 46 low severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
```

### Fix compile errors
```sh
$ npm run build

> k2informaticsgmbh.github.io@1.0.0 build ...\k2informaticsgmbh.github.io\dev
> webpack --progress --colors --config webpack.prod.js

...\k2informaticsgmbh.github.io\dev\node_modules\webpack-cli\bin\config-yargs.js:89
                                describe: optionsSchema.definitions.output.properties.path.description,
                                                                           ^

TypeError: Cannot read property 'properties' of undefined
    at module.exports (...\k2informaticsgmbh.github.io\dev\node_modules\webpack-cli\bin\config-yargs.js:89:48)
    at ...\k2informaticsgmbh.github.io\dev\node_modules\webpack-cli\bin\webpack.js:60:27
    at Object.<anonymous> (...\k2informaticsgmbh.github.io\dev\node_modules\webpack-cli\bin\webpack.js:515:3)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! k2informaticsgmbh.github.io@1.0.0 build: `webpack --progress --colors --config webpack.prod.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the k2informaticsgmbh.github.io@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     ...\AppData\Roaming\npm-cache\_logs\2018-10-01T11_30_35_183Z-debug.log
```

I will continue my review after these are fixed!

**Comment by shamis** [comment]
@janmenjoyroy, I am also getting the same error as @c-bik .
node version is 8.11.1
npm version is 6.3.0
Please let me know if there some other parameter has to be checked.

**Comment by c-bik** [comment]
@shamis it compiles now and `npm start` alos works for quickly checking the progress. Can you please verify if your review comments are addressed as you expected?

**Comment by c-bik** [comment]
@janmenjoyroy can you please have a look at review comments from @shamis above. Please also lowercase all files extensions for consistancy

I don't see the following translations files

image

being copied to `public`

image

**we need them in `public` in order to change the language specific page translations live (WITHOUT `npm run build`)**

Instead of
```sh
locales
|-- de
|   `-- translation.json
`-- en
    `-- translation.json
```
can it be
```sh
translations
|-- de.json
|-- it.json
|-- fr.json
`-- en.json
```

**Comment by c-bik** [comment]
image
Change to **K2 Informatics GmbH**

**Review by c-bik — CHANGES_REQUESTED** [review]
image
Change to **K2 Informatics GmbH**

can you please have a look at review comments from @shamis above. Please also lowercase all files extensions for consistancy

I don't see the following translations files

image

being copied to `public`

image

**we need them in `public` in order to change the language specific page translations live (WITHOUT `npm run build`)**

Instead of
```sh
locales
|-- de
|   `-- translation.json
`-- en
    `-- translation.json
```
can it be
```sh
translations
|-- de.json
|-- it.json
|-- fr.json
`-- en.json
```

**Review by c-bik — APPROVED** [review]

**Review by c-bik — CHANGES_REQUESTED** [review]
Ref: https://github.com/K2InformaticsGmbH/k2informaticsgmbh.github.io/pull/1#pullrequestreview-164243204
1. Company name correction at footer at footer isn't fixed!
1. Changing translation in public/translations/_language_.json doesn't load on page reload
1. URL should change based on laguage selection (default _for now_ `en`). E.g. `http://localhost:8082/en` etc

**Comment by janmenjoyroy** [comment]
@c-bik I am yet to fix:
1. Changing translation in public/translations/language.json doesn't load on page reload
2. URL should change based on laguage selection (default for now en). E.g. http://localhost:8082/en etc

**Comment by janmenjoyroy** [comment]
Fixed all comments; @c-bik @shamis please review and let me know in case of any issue.

**Comment by c-bik** [comment]
> Fixed all comments; @c-bik @shamis please review and let me know in case of any issue.

Please also address the following:
image

image

File: dev/assets/magnific-popup/magnific-popup.js [hunk 1]
```diff
@@ -0,0 +1,1860 @@
+/*! Magnific Popup - v1.1.0 - 2016-02-20
```
Comment (c-bik) [inline dev/assets/magnific-popup/magnific-popup.js]: Is it used?
Can it be imported through `npm` instead, if required?

File: dev/package-lock.json [hunk 2]
```diff
@@ -27,357 +27,215 @@
       "dev": true
     },
     "@fortawesome/fontawesome-common-types": {
-      "version": "0.2.2",
-      "resolved": "https://registry.npmjs.org/@fortawesome/fontawesome-common-types/-/fontawesome-common-types-0.2.2.tgz",
-      "integrity": "sha512-OBzXzYJ+nDMece2nKeuV7hLuwqPN3jv1Fk56gag2DL4BiwWl8gkFQplj5Krep9HwtjFxbM6/DWN6ZG7FRC5N0w=="
+      "version": "0.2.6",
```
Comment (c-bik) [inline dev/package-lock.json]: Delete this from repo, and list in .gitignore!
No need to version lock files when developing

File: dev/assets/js/main.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
+
+$(document).ready(function ($) {
```
Comment (c-bik) [inline dev/assets/js/main.js]: Use shorthad construct
```js
$(function ...
```
See `index.js`

**Review by c-bik — CHANGES_REQUESTED** [review]
1. Delete `dev/assets/magnific-popup` (if needed use `npm` to import it)
1. Move `dev/assets/js` to `dev/js`
1. Delete any unused assets (except the CI folder)

**Comment by c-bik** [comment]
@janmenjoyroy 
```bash
Bikram@WKS006 MINGW64 /c/projects/git/K2InformaticsGmbH/k2informaticsgmbh.github.io/dev (janmenjoyroy-development)
$ npm run build

> k2informaticsgmbh.github.io@1.0.0 build C:\projects\git\K2InformaticsGmbH\k2informaticsgmbh.github.io\dev
> webpack --progress --colors --config webpack.prod.js

'webpack' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! k2informaticsgmbh.github.io@1.0.0 build: `webpack --progress --colors --config webpack.prod.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the k2informaticsgmbh.github.io@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Bikram\AppData\Roaming\npm-cache\_logs\2018-10-26T06_49_47_968Z-debug.log
```

@shamis can you please verify/confirm? _my other npm projects are compiling fine_

**Comment by c-bik** [comment]
@janmenjoyroy

**looks like you keep missing this one! (been reported by me and @shamis above)**

image

Please fix!

**Comment by acautin** [comment]
@c-bik for me it is building fine after clean clone on OS X, are you sure your ``npm install`` completed successfully ?

**Comment by c-bik** [comment]
> @c-bik for me it is building fine after clean clone on OS X, are you sure your `npm install` completed successfully ?

Didn't try a clean clone! Will try shortly and report back soon (thanks for headsup 👍 )!

**Comment by c-bik** [comment]
@janmenjoyroy 
image

This needs to work!

**Comment by janmenjoyroy** [comment]
> @janmenjoyroy
> image
> 
> This needs to work!

@c-bik and @shamis, I need your help in resolving this. Please guide me on how to fix this type of issue.

**Comment by acautin** [comment]
@janmenjoyroy @c-bik I haven't tested it but think the problem is the ajax request are not supported when using the file:// protocol, to test it you need add it to a server for it to work via the http:// protocol.

**Comment by c-bik** [comment]
> @janmenjoyroy @c-bik I haven't tested it but think the problem is the ajax request are not supported when using the file:// protocol, to test it you need add it to a server for it to work via the http:// protocol.

@acautin 👍 So my test was flawed. I think this issue doesn't exist with `npm start` (_I will verify again and confirm_).

I, however, am quite sure that even with `npm start` (i.e. running a webpack server) the live translation change still doesn't work! (@janmenjoyroy can you please confirm?)

**Comment by janmenjoyroy** [comment]
Hi Agustin,
Nice to meet you !!
Just to keep you updated, we are working on the UI development of the new website for K2i.
We have built the web pages to support multiple languages. We use AJAX to load the language JSON files.This features is working fine using a local server. But we face the said issue when we load this webpage statically (in a browser); basically we open it without any server.
We get the following error "failed to load file://...../locals/en/translation.json", when we try to open the page statically.
Most of the blogs suggested either to use a local web server or upload the language file to another server.(viz. "Cross origin requests are only supported for HTTP." error when loading a local file)

| 
| 
| 
|  |  |

 |

 |
| 
|  | 
"Cross origin requests are only supported for HTTP." error when loading ...

I'm trying to load a 3D model into Three.js with JSONLoader, and that 3D model is in the same directory as the e...
 |

 |

 |




I am stuck with this issue for sometime and it will be really helpful if you please suggest me some way out for this issue.
Regards, Janmenjoy Roy 
 

    On Thursday, 8 November, 2018, 3:06:59 PM IST, Bikram Chatterjee <notifications@github.com> wrote:  
 
 

@janmenjoyroy @c-bik I haven't tested it but think the problem is the ajax request are not supported when using the file:// protocol, to test it you need add it to a server for it to work via the http:// protocol.


@acautin 👍 So my test was flawed. I think this issue doesn't exist with npm start (I will verify again and confirm).

I, however, am quite sure that even with npm start (i.e. running a webpack server) the live translation change still doesn't work! (@janmenjoyroy can you please confirm?)

**Comment by acautin** [comment]
Well I think it will never work locally as the file:// protocol is not supported, you should test if the translations work when using a local test server, for example with ``npm start``. If that works then it should be fine.

**Comment by janmenjoyroy** [comment]
> > @janmenjoyroy @c-bik I haven't tested it but think the problem is the ajax request are not supported when using the file:// protocol, to test it you need add it to a server for it to work via the http:// protocol.
> 
> @acautin 👍 So my test was flawed. I think this issue doesn't exist with `npm start` (_I will verify again and confirm_).
> 
> I, however, am quite sure that even with `npm start` (i.e. running a webpack server) the live translation change still doesn't work! (@janmenjoyroy can you please confirm?)

@c-bik, regarding your last query, I have carried out the following steps and found that live translation changes get reflected on browser reload. I have carried out the test with a fresh cloned code base.
1. run *npm install* and then *npm start* (the home page gets opened automatically)
2. change the language JSON file
3.  reload the browser
4. verify the changes in the web page

Is my understanding correct? Or did I miss out any step?

**Comment by c-bik** [comment]
> > > @janmenjoyroy @c-bik I haven't tested it but think the problem is the ajax request are not supported when using the file:// protocol, to test it you need add it to a server for it to work via the http:// protocol.
> > 
> > 
> > @acautin 👍 So my test was flawed. I think this issue doesn't exist with `npm start` (_I will verify again and confirm_).
> > I, however, am quite sure that even with `npm start` (i.e. running a webpack server) the live translation change still doesn't work! (@janmenjoyroy can you please confirm?)
> 
> @c-bik, regarding your last query, I have carried out the following steps and found that live translation changes get reflected on browser reload. I have carried out the test with a fresh cloned code base.
> 
> 1. run _npm install_ and then _npm start_ (the home page gets opened automatically)
> 2. change the language JSON file
> 3. reload the browser
> 4. verify the changes in the web page
> 
> Is my understanding correct? Or did I miss out any step?

@janmenjoyroy Thanks 👍 I will confirm tomorrow!

**Comment by c-bik** [comment]
> > I, however, am quite sure that even with `npm start` (i.e. running a webpack server) the live translation change still doesn't work! (@janmenjoyroy can you please confirm?)
> 
> @c-bik, regarding your last query, I have carried out the following steps and found that live translation changes get reflected on browser reload. I have carried out the test with a fresh cloned code base.
> 
> 1. run _npm install_ and then _npm start_ (the home page gets opened automatically)
> 2. change the language JSON file
> 3. reload the browser
> 4. verify the changes in the web page
> 
> Is my understanding correct? Or did I miss out any step?

@janmenjoyroy Your understanding is correct, however when I follow the excat same steps as you understood I see the problem Step 4 (in your steps above) fails!

Here is what is happening to me...
image

@shamis / @acautin can you please confirm?

PS: @janmenjoyroy I fixed the other two issues you were stuck with (extension lower cased and `npm audit fix`) and committed my changes to your fork so that the PR is updated and merge conflict is now gone. Please pull into your setup.

**Review by c-bik — CHANGES_REQUESTED** [review]
### COSMETICS and CODE Cleanup
1. Remove all unused files:
For example *.psd files are shown modified in PR
image **is however _NOT_ referenced anywhere in code**
image _few other unused files are with extenions eps, pdf, ai, svg etc_

2. Remove this change from PR
image

### IMPORTANT
Additionaly, @agustin and @shamis can you please do a coding, architecture, best practices review for @janmenjoyroy with improvement possibilities

**Review by c-bik — APPROVED** [review]

File: dev/webpack.common.js [hunk 4]
```diff
@@ -31,18 +31,27 @@ module.exports = {
...
         }),
         new HtmlWebpackPlugin({
-            title: pkg.description + ' ' + pkg.version
+            title: pkg.description + ' ' + pkg.version,
+            template: './html/template.html',
+            favicon:'./assets/images/logo.png',
+            inject:'head'
         }),
-        new favicon('./assets/images/logo.png')
+        // new favicon('./assets/images/logo.png')
```
Comment (c-bik) [inline dev/webpack.common.js]: remove commented lines

File: dev/webpack.common.js [hunk 5]
```diff
@@ -31,18 +31,27 @@ module.exports = {
...
     ],
     module: {
         rules: [
             {
-                test: /\.(scss)$/,
-                use: ['style-loader', 'css-loader', 'postcss-loader', 'sass-loader']
+                test: /\.(scss|css)$/,
+                use: ['style-loader', 'css-loader','sass-loader']
+                //test: /\.(scss)$/,
+                // use: ['style-loader', 'css-loader', 'postcss-loader', 'sass-loader']
```
Comment (c-bik) [inline dev/webpack.common.js]: remove commented lines

File: dev/webpack.dev.js [hunk 6]
```diff
@@ -6,7 +6,7 @@ module.exports = merge(common, {
     mode: "development",
     devtool: 'inline-source-map',
     devServer: {
-        contentBase: '../public'
+         contentBase: '../public'
```
Comment (c-bik) [inline dev/webpack.dev.js]: revert indent (extra space)

File: dev/index.js [hunk 7]
```diff
@@ -1,73 +1,274 @@
...
+  about_us: require("./html/fragments/about_us.html")
 };
 
-$(function () {
-    //trimHtmlFragments();
+function loadHeader() {
+  let headerElem = '<meta charset="utf-8">';
+  headerElem = `${headerElem} <meta content="width=device-width, initial-scale=1.0" name="viewport">`;
+  headerElem = `${headerElem} <meta content="" name="keywords">`;
+  headerElem = `${headerElem} <meta content="" name="description">`;
```
Comment (c-bik) [inline dev/index.js]: Do not inject HTML string. Construct shadow dom with jquery instead

File: dev/index.js [hunk 8]
```diff
@@ -1,73 +1,274 @@
...
+    if (i === 1) {
+      let ulElem = $("<ul/>");
+      for (let j = 0; j < 5; j++) {
+        let li = $("<li/>");
+        li.append(
+          "<i class='fas fa-angle-double-right'></i>&nbsp;<span data-i18n='[html]about_us.vision.contents." +
+            i +
+            "." +
+            j +
+            "'></span>"
```
Comment (c-bik) [inline dev/index.js]: Do not inject HTML string, use jQuery to construct the dom to append.

File: dev/index.js [hunk 9]
```diff
@@ -1,73 +1,274 @@
...
+    let innerDiv1 = $("<div/>").addClass("member");
+    let innerDiv2 = $("<div/>").addClass("pic");
+    let img = $("<img/>")
+      .attr({
+        src: "../../assets/team/portrait_" + i + ".jpg"
+      })
+      .css({
+        width: "100px",
+        height: "100px"
+      });
```
Comment (c-bik) [inline dev/index.js]: reduce number of lines if it fits within 80 characters!

**Review by c-bik — CHANGES_REQUESTED** [review]
Code can be quite improved / rediced using some templating (like handlebars) and keeping styling out of JS code

**Comment by janmenjoyroy** [comment]
hi @acautin and @c-bik will you please review the code and provide me with your valuable feedback?
I have pushed all the changes requested by @c-bik.

File: dev/index.js [hunk 10]
```diff
@@ -1,73 +1,291 @@
...
-            }
-        })
-        .resolve();
-    router.updatePageLinks();
-    console.log(process.env.NODE_ENV);
+$(function () {
+  $(document.head).append(loadHeader());
+  router
+    .on({
+      "/": function () {
```
Comment (c-bik) [inline dev/index.js]: can you redirect this to `/:lang?/home` instead?

File: dev/index.js [hunk 11]
```diff
@@ -1,73 +1,291 @@
...
+        });
+        li.append(icon);
+        li.append("&nbsp;");
+        li.append(span);
+        ulElem.append(li);
+      }
+      $("#visionList").append(ulElem);
+    } else {
+      $("#visionList").append(
+        "<p data-i18n='[html]about_us.vision.contents." + i + "'></p>"
```
Comment (c-bik) [inline dev/index.js]: rewrite with `$('<p>').attr()` etc

File: dev/index.js [hunk 12]
```diff
@@ -1,73 +1,291 @@
...
+      .attr({
+        src: "../../assets/team/portrait_" + i + ".jpg"
+      })
+      .css({width: "100px", height: "100px"});
+    innerDiv2.append(img);
+    innerDiv1.append(innerDiv2);
+    innerDiv1.append(
+      '<h5 style="font-size:16px" data-i18n="[html]about_us.teams.list.' +
+      i +
+      '.name"></h5>'
```
Comment (c-bik) [inline dev/index.js]: rewrite `$('<h5>').attr(...)`

File: dev/index.js [hunk 13]
```diff
@@ -1,73 +1,291 @@
...
+    let innerDiv2 = $("<div/>").addClass("pic");
+    let img = $("<img/>")
+      .attr({
+        src: "../../assets/team/portrait_" + i + ".jpg"
+      })
+      .css({width: "100px", height: "100px"});
+    innerDiv2.append(img);
+    innerDiv1.append(innerDiv2);
+    innerDiv1.append(
+      '<h5 style="font-size:16px" data-i18n="[html]about_us.teams.list.' +
```
Comment (c-bik) [inline dev/index.js]: use bootstrap class instead of style inlining wherever possible

File: dev/index.js [hunk 14]
```diff
@@ -1,73 +1,291 @@
...
+    let td1 = $("<td />")
+      .css({width: "30%", height: "40px"})
+      .addClass("text-center align-middle")
+      .attr({
+        "data-i18n": "[html]about_us.contact.contents." + i + ".key"
+      });
+    tr.append(td1);
+    /* For Second Row */
+    let td2 = $("<td />")
+      .css({height: "40px"})
```
Comment (c-bik) [inline dev/index.js]: how does a hardcoded height becomes responsive?

File: dev/index.js [hunk 15]
```diff
@@ -1,73 +1,291 @@
...
+    tbody.append(tr);
+  }
+  table.append(tbody);
+  $("#contactList").append(table);
+  /* End Contact section */
+
+  /* Strat support section */
+  for (let i = 0; i < 4; i++) {
+    $("#supportContents").append(
+      '<span data-i18n="[html]about_us.support.contents.' + i + '"></span><br/>'
```
Comment (c-bik) [inline dev/index.js]: rewrite with jq

File: dev/index.js [hunk 16]
```diff
@@ -1,73 +1,291 @@
...
+  /* End support section */
+
+  /* Strat jobs Section */
+  for (let i = 0; i < 12; i++) {
+    if (i === 5) {
+      let ulElem = $("<ul />");
+      for (let j = 0; j < 5; j++) {
+        let li = $("<li />");
+        li.append(
+          "<i class='fas fa-angle-double-right'></i>&nbsp;<span data-i18n='[html]about_us.jobs.contents." +
```
Comment (c-bik) [inline dev/index.js]: **rewrite all hard coded HTML fragment strings with jQuery EVERYWHERE** (was already the earlier review comment)

**Review by c-bik — CHANGES_REQUESTED** [review]

File: dev/index.js [hunk 17]
```diff
@@ -67,12 +65,7 @@ $(function () {
     .on({
       "/": function () {
         console.log("Home");
-        $("#main")
-          .empty()
-          .append(htmlFrags.index);
-        i18next.changeLanguage(window.lang);
-        jqueryI18next.init(i18next, $);
-        $("body").localize();
+        router.navigate('/en/home');
```
Comment (acautin) [inline dev/index.js]: is this always going to english ? should it not be `router.navigate(window.lang + '/home')` ?

**Comment by acautin** [comment]
@janmenjoyroy sure I will, I need to discuss the goals of the project with @stoch  as I was not involved at the beginning.

**Comment by acautin** [comment]
@janmenjoyroy I have reviewed a little bit the code, for me it seems there is in good state good job from you and @c-bik checking. But there are still a couple of details that need fixing:

The alignment on windows between 780 and 890 pixels wide is not proper:

screen shot 2018-12-14 at 10 04 23

The sandwich menu for mobile screens is also not working (translate or sections):

screen shot 2018-12-14 at 10 05 33

**Comment by c-bik** [comment]
@acautin FYI:
https://github.com/K2InformaticsGmbH/k2informaticsgmbh.github.io/tree/master hosts for https://k2informaticsgmbh.github.io/

https://github.com/K2InformaticsGmbH/k2informaticsgmbh.github.io/tree/master/erloci_dev_setup is temporary home for from erloci docs @KarlKeiser generated (we will move it away from here into wiki eventually)!

**Comment by janmenjoyroy** [comment]
@acautin I have fixed your following review comments:
1. The alignment on windows between 780 and 890 pixels wide is not proper
2. The sandwich menu for mobile screens is also not working (translate or sections)
Please review the same and let me know in case of any issue.

**Comment by acautin** [comment]
@janmenjoyroy please do not comment out code, if is not usable anymore just remove it. It can still be retrieved by the git history and it is easier to see the difference.
There is duplicated code on the functions you added please extract into a separated function and call it instead.

**Comment by janmenjoyroy** [comment]
@acautin @c-bik ... Happy New Year ...
Are there any other change requests from your end for the k2i website?

**Review by c-bik — APPROVED** [review]

**Comment by acautin** [comment]
@janmenjoyroy thanks for your effort and the patience will merge now, to have this as our initial version of the page, we have some improvements that are going to come but in general if you want to think about how it could be implemented, changes are regarding making the team sections easier to modify and a better highlight of our current project.

**Comment by janmenjoyroy** [comment]
@acautin @c-bik thank you for all your support.

@acautin can I have your Skype ID please? We can discuss regarding the improvisation of the team page at your convenience. And you can reach me at Skype: janmenjoyroy.

**Comment by acautin** [comment]
@janmenjoyroy We don't have the plan yet, will discuss internally what are the goal and suggest a couple of alternatives during next week and will come back to you for a meeting, my initial idea would be some sort of json file from which the data can be loaded but is not yet finalized.

**Comment by acautin** [comment]
@janmenjoyroy one unrelated small thing. Can you please check your ``git config --list`` and verify ``core.ignorecase=true`` if you use a windows or case insensitive Os X. Otherwise ``core.ignorecase=false`` for linux based systems. As to avoid including multiple versions of the same file with differences only in the casing. Thanks!

**Pull request merged**